sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -58,9 +58,7 @@ static inline std::string codeToString(cl_int code){
 {                                                                              \
   auto code = expr;                                                            \
   if (code != CL_SUCCESS) {                                                    \
-    std::string errorMessage(OCL_ERROR_REPORT + codeToString(code));           \
```

Comment:
We could wrap this code by `#ifdef` - so anyone can select what he want

Reply:
Why do you want to emit exception message to `std::cerr`?

Reply:
It was agreed that adding a default async handler separately would be a _much_ better long-term solution.

Reply:
@bader, if it is too hard for me to update existing SYCL code to insert async exceptions handler

Reply:
@AlexeySachkov, this change has nothing to do with async exceptions.

Reply:
Please, disregard my comment then